### PR TITLE
Natural language coeff

### PR DESCRIPTION
**Pull request by lilian-sanselme-prl**
This Pull Request adds a coefficient to correct for the fact that english language contains letters that repeat themselves.
The diff on file `docs/images/swap_light.png` explains the result quite well.

### CODE REVIEW

**Review by felix-martel-prl — COMMENTED** [review]
Can you mention in the docs the corpus you used for calibrating this coefficient? I assume it's not the same corpus you use in your other experiments :innocent: 

Otherwise LGTM!

File: docs/swap_unbiasing.md [hunk 1]
```diff
@@ -321,5 +321,12 @@ As mentioned earlier, there is a maximum reachable in the Character Error Rate w
 This maximum is directly due to the constraint of preventing swapping the current character if it has already been swapped.
 
 
+## Taking into account repetitions of characters in natural language
+
+In natural language, our assumption that two consecutive characters are always different is not always true.
+To take this into account, we have to introduce a small multiplicative coefficient to the probability $p$.
+
+For english language, we computed this correction factor to be approximately 1.0655, which is the default in `textnoisr`.
```
Comment (felix-martel-prl) [inline docs/swap_unbiasing.md:329]: nit:You may want to explain how to compute this coefficient for other languages. Also it's _English_, with a capital letter.

**Comment by lilian-sanselme-prl** [comment]
Thanks for the review!

I added a script to compute this figure. I use `glue-mnli` now, a different dataset than the one used for the tests or the calibration figures.

**Review by felix-martel-prl — APPROVED** [review]
LGTM!

IMO the next step would be to pre-compute this coefficient for a range of common language (`CharAugmenter(language="en")` is better than `CharAugmenter(natural_language_swap_correction=1.052)`).

**Pull request merged**